src/Akeneo/Bundle/ElasticsearchBundle/Client.php

```diff
@@ -66,6 +67,38 @@ public function index($indexType, $id, array $body)
 
     /**
      * @param string $indexType
+     * @param array  $documents
+     * @param string $keyAsId
+     *
+     * @throws MissingIdentifierException
+     *
+     * @return array see {@link https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_indexing_documents.html#_bulk_indexing}
+     */
+    public function bulkIndexes($indexType, $documents, $keyAsId)
```

Comment:
You could also call it learn (in the idea of wrapping the library) but this naming is fine.